services/src/atdd-staging/src/main/resources/features/failsafe_behaviour.feature

```diff
@@ -10,9 +10,10 @@ Feature: Failsafe Suite
     Given Create 1 flow with A Switch used and at least 1 alternate path between source and destination switch and 500 bandwidth
 
     When ISL between switches loses connectivity
-    And remains in this state for 30 seconds
+    And remains in this state for 240 seconds
```

Comment:
I think it is better to update our staging config and set a lesser timeout. Why wait for 240 seconds if it is configurable